### PR TITLE
need ExpectNoError check

### PR DESCRIPTION
**Pull request by foxyriver**
**What this PR does / why we need it**:

err need ExpectNoError check

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/metrics_util.go [hunk 1]
```diff
@@ -543,6 +543,7 @@ func sendRestRequestToScheduler(c clientset.Interface, op string) (string, error
 func getSchedulingLatency(c clientset.Interface) (*SchedulingMetrics, error) {
 	result := SchedulingMetrics{}
 	data, err := sendRestRequestToScheduler(c, "GET")
+	ExpectNoError(err)
```
Comment (shyamjvs) [inline test/e2e/framework/metrics_util.go]: I don't think you should be triggering failure here. It's better to instead `return &result, err`.

Comment (shyamjvs) [reply test/e2e/framework/metrics_util.go]: So that the error can be handled by the caller of this function.

**Comment by neolit123** [comment]
/sig testing
^ ?

**Comment by foxyriver** [comment]
@shyamjvs PTAL

**Pull request merged**